t/t-standalone-file.sh

```diff
@@ -64,6 +64,67 @@ do_upload_download_test () {
   [ "$(echo "$objectlist" | wc -l)" -eq 12 ]
 }
 
+do_local_path_test () {
```

Comment:
I might suggest having this function take `$reponame` as an argument.  I suspect this still works as-is because variables are global in the shell by default, so the caller can set `$reponame` and then this function can just read it below.  But it would be a bit more future-proof to maybe have it take the variable as an argument and set a local variable from it.

Reply:
That sounds like a good suggestion.  I've squashed in changes to that effect here and below.